assets/js/app.js

```
@@ -23,7 +23,10 @@ import {LiveSocket} from "phoenix_live_view"
 import topbar from "../vendor/topbar"
 
 // Define Phoenix Hooks
+import InfiniteScroll from "./hooks/infinite_scroll"
+
 let Hooks = {}
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'Hooks' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)